docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -134,35 +223,16 @@ you can force the broker to reload images from your org.
 apb bootstrap
 ----
 
-== Using the mobile-service tag
-
-For MCP enabled _APBs_, make sure we have a `mobile-service` tag inside the `apb.yml` file:
-
-....
-...
-tags: 
-  - mobile-service
-....
-
-== Using the servicename label
-
-Inside of the `apb.yml` file, make sure you use the `serviceName:` label, like:
-
-....
-...
-metadata:
-  displayName: Aerogear Sync Server
-  console.openshift.io/iconClass: icon-nodejs
-  serviceName: fh-sync-server
-...
-....
-
 == Testing an APB
 
-Currently, there are not many tools for testing. The APB team has a few open Github issues and proposals:
+Good practice is to have a special playbook for testing, called ``test.yml``. This playbook is used for quick verification of the 
```

Comment:
I think this is not only good practice but the intended way to test an APB: running `apb test` will look for the test playbook and execute the roles in there.

Reply:
Good practice is to include a test.yml, not all developers would, so saying that it is good practice I think makes sense. 

An APB without a test.yml but with every other requirement would function as a custom mobile service a test.yml is just good practice.

Reply:
If I link to the APB docs on testing I think that should clear it up?

Reply:
Yes, that should be fine. No need to go into the details here.

Reply:
Turns out I'm already linking to the APB docs on testing right below this.